plugins/plugin-client-default/notebooks/dashboard.md

```diff
@@ -0,0 +1,31 @@
+---
+layout:
+  1: default
+  2: default
+  3: default
+---
+
+```bash
+---
+execute: now
+---
+ray job info raysubmit_mP8gXdLKVTkdEwfX
+```
+
+---
+
+```bash
+---
+execute: now
+---
+ray job logs raysubmit_mP8gXdLKVTkdEwfX
+```
+
+---
+
+```bash
```

Comment:
@PabloCarmona I think we can remove this section for now till we have this implemented.